index.js

```diff
@@ -1,15 +1,24 @@
 /* eslint-disable no-console */
 import fs, { copyFileSync } from "fs";
+import { getBuildInfo } from '@netlify/build-info/node';
 
 const SITE_ID = "321a7119-6008-49a8-9d2f-e20602b1b349";
 
+async function projectUsesNextJS() {
+  for (const framework of (await getBuildInfo()).frameworks) {
+    if (framework.id == 'next') {
```

Comment:
nit: Strict comparison for consistency.

```suggestion
    if (framework.id === 'next') {
```